include/af/arith.h

```diff
@@ -741,6 +741,17 @@ extern "C" {
     */
     AFAPI af_err af_not   (af_array *out, const af_array in);
 
+    /**
```

Comment:
#if AF_API_VERSION >= 38

Reply:
I don't understand what is different from the file and your comment ?